content/tutorials/manage-the-rippled-server/troubleshooting/health-check-interventions.md

```diff
@@ -0,0 +1,104 @@
+
+## Suggested Interventions
+
+When a health check fails, and it's not just a [momentary failure](#momentary-failures), the action to take to recover from the outage varies based on the cause. You may be able to configure your infrastructure to fix some types of failures automatically. Other failures require the intervention of a human administrator who can investigate and take the necessary steps to resolve more complex or critical failures; depending on the structure of your organization, you may have different levels of human administrator so that less skilled, lower level administrators can fix certain issues independently, but need to escalate to higher level administrators to fix larger or more complex issues. How and when you respond is likely to depend on your unique situation, but the metrics reported in the health check result can be a factor in these decisions.
+
+The following sections suggest some common interventions you may want to attempt and the health check statuses most likely to prompt those interventions. Automated systems and human administrators may selectively escalate through these and other interventions:
+
+- [Redirect traffic](#redirect-traffic) away from the affected server
+- [Restart](#restart) the server software or hardware
+- [Investigate network](#investigate-network) in case the problem originates elsewhere
+- [Replace hardware](#replace-hardware)
+- [Upgrade](#upgrade) the `rippled` software
+
+
+### Redirect Traffic
+
+A common reliability technique is to run a pool of redundant servers through one or more load-balancing proxies. You can do this with `rippled` servers, but should not do this with [validators](rippled-server-modes.html). In some cases, the load balancers can monitor the health of servers in their pools and direct traffic only to the servers that are currently reporting themselves as healthy. This allows servers to recover from being temporarily overloaded and automatically rejoin the pool of active servers.
+
+Redirecting traffic away from a server that is unhealthy is an appropriate response, especially for servers that report a `health` status of `warning`. Servers in the `critical` range may need more significant interventions.
+
+
+### Restart
+
+The most straightforward intervention is to restart the server. This can resolve temporary issues with several types of failures, including any of the following metrics:
+
+- `load_factor`
```

Comment:
what do these metrics mean? see my above comment about a doc that lists/explains the metrics that the method checks

Reply:
Linked the word "metrics" to the table of the metrics and their definitions